cmd/keeper/keeper.go

```diff
@@ -1754,3 +1757,63 @@ func keeper(c *cobra.Command, args []string) {
 
 	lockFile.Close()
 }
+
+func (p *PostgresKeeper) generateHBA(cd *cluster.ClusterData, role common.Role, userHba []string) []string {
+
+	// Minimal entries for local normal and replication connections needed by the stolon keeper
+	// Matched local connections are for postgres database and suUsername user with md5 auth
+	// Matched local replication connections are for replUsername user with md5 auth
+	computedHBA := []string{
+		fmt.Sprintf("local postgres %s %s\n", p.pgSUUsername, p.pgSUAuthMethod),
+		fmt.Sprintf("local replication %s %s\n", p.pgReplUsername, p.pgReplAuthMethod),
+	}
+
+	// when no custom hba is set, we default to open to all
+	if len(userHba) == 0 {
+		computedHBA = append(
+			computedHBA,
+			"host all all 0.0.0.0/0 md5",
+			"host all all ::0/0 md5",
+		)
+	}
```

Comment:
this must be at the end. After the stolon generated rules.

Reply:
Done